framework/src/Volo.Abp.AspNetCore.Mvc.UI.Theme.Shared/wwwroot/libs/abp/aspnetcore-mvc-ui-theme-shared/jquery/jquery-extensions.js

```diff
@@ -35,7 +35,12 @@
                 //change text
                 if ($buttonInnerSpan.length && $button.attr('data-busy-text')) {
                     $button.data('buttonOriginalText', $buttonInnerSpan.html());
-                    $buttonInnerSpan.html($button.attr('data-busy-text'));
+                    
+                    if ($button.data('busy-text-is-html')) {
```

Comment:
Is this should be `data-busy-text-is-html`?

Reply:
I think `busy-text-is-html` is correct here, since I use jQuerys `data()`-function:

```javascript
var element = $('<button>').attr('data-busy-text-is-html', true);
» undefined

element.data('busy-text-is-html');
» true

element.attr('data-busy-text-is-html')
» "true"
```

Reply:
You are right, Thanks.